tools/clang/test/CMakeLists.txt

```diff
@@ -148,5 +154,12 @@ if (WIN32)
             DEPENDS ExecHLSLTests dxexp
             ARGS ${CLANG_TEST_EXTRA_ARGS}
           )
+
+  add_lit_target("check-clang-hash-stability" "Running lit suite dxc hash stability test"
```

Comment:
We shouldn't need to add a target here explicitly, the `add_lit_testsuites` call above adds targets for each subdirectory.

Reply:
Removed.